dev/src/External/CashierDesk.java

```diff
@@ -66,21 +64,28 @@ public void turnOn() throws Exception {
                     break;
                 }
             }
-
         }
     }
 
-    private boolean chooseStore() {
-        this.out.println("There are: " + this.controllerRef.getStores().size() + " stores");
-        this.out.print("Please enter store id: ");
-        int storeIndex = Integer.parseInt(this.in.nextLine());
-        if (storeIndex < this.controllerRef.getStores().size())
-            chosenStore = this.controllerRef.getStores().get(storeIndex);
-        else {
-            this.out.println("Invalid store id\nReturning to main menu...\n");
+
+    private boolean displayStores() {
```

Comment:
לא ראיתי שמשתמשים בערך הבוליאני שחוזר בשום מקום, הוא מיותר אז

Reply:
Fixed in #50